react/src/components/organisms/Header/index.js

```diff
@@ -297,8 +297,7 @@ Header.defaultProps = {
       width: 45,
       height: 45
     },
-    siteName: 'Mass.gov',
-    title: 'Mass.gov homepage'
+    siteName: 'Mass.gov'
```

Comment:
```suggestion
    siteName: 'Mass.gov',
    title: 'Go to Mass.gov Homepage',
```

Reply:
let's put back this for now, we can come back to this one and Footer defaultProps together